Src/VimCore/CoreInterfaces.fs

```diff
@@ -2570,6 +2611,14 @@ type TextObjectKind =
     | AlwaysCharacter
     | AlwaysLine
 
+/// The kind of visual insertion to perform
+[<RequireQualifiedAccess>]
+[<NoComparison>]
+type VisualInsertKind =
+    | Start
+    | End
+    | EndOfLine

```

Comment:
Can we add a comment that explains the difference between `End` and `EndOfLine`? Not sure it's obvious at a glance.